osgp/platform/osgp-core/src/main/java/org/opensmartgridplatform/core/application/services/DeviceResponseMessageService.java

```diff
@@ -192,7 +191,7 @@ private void handleProtocolResponseMessage(final ProtocolResponseMessage message
 
     private ProtocolRequestMessage createProtocolRequestMessage(final ProtocolResponseMessage message)
             throws JMSException {
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove the declaration of thrown exception 'javax.jms.JMSException', as it cannot be thrown from method's body. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://ci.opensmartgridplatform.org/sonarqube/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)